docs/Spark/gems/source-target/file/iceberg.md

```diff
@@ -0,0 +1,193 @@
+| Table Name                  | Name of the table                                                                                                                                                           | True                                                                         |
+| File Location               | External file path to store data (Only applicable if Catalog type is Hive.)                                                                                                 | False                                                                        |
+| Partition Columns           | List of columns to partition the Iceberg table by (Provide it during createOrReplace write mode to leverage overwritePartitions write mode in future.)                      | False                                                                        |
+| Merge schema                | If true, then any columns that are present in the DataFrame but not in the target table are automatically added on to the end of the schema as part of a write transaction. | False                                                                        |
+
+#### Supported Write Modes
+
+| Write Mode | Description                                                                                                                      |
+| ---------- | -------------------------------------------------------------------------------------------------------------------------------- |
+| overwrite  | If data already exists, overwrite with the contents of the DataFrame                                                             |
+| append     | If data already exists, append the contents of the DataFrame                                                                     |
+| ignore     | If data already exists, do nothing with the contents of the DataFrame. This is similar to a `CREATE TABLE IF NOT EXISTS` in SQL. |
+| error      | If data already exists, throw an exception.                                                                                      |
+
+Among these write modes overwrite and append works the same way as in case of parquet file writes.
+
+### Target Example
+
+<div class="wistia_responsive_padding" style={{padding:'56.25% 0 0 0', position:'relative'}}>
+<div class="wistia_responsive_wrapper" style={{height:'100%',left:0,position:'absolute',top:0,width:'100%'}}>
+<iframe src="https://fast.wistia.net/embed/iframe/8j80hc4k1e?seo=false?videoFoam=true" title="Iceberg-target-example" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" msallowfullscreen width="100%" height="100%"></iframe>
+</div></div>
+<script src="https://fast.wistia.net/assets/external/E-v1.js" async></script>
+
+### Generated Code {#target-code}
+
+````mdx-code-block
+
+<Tabs>
+
+<TabItem value="py" label="Python">
+
+```py
```

Comment:
Same here, is it possible to get the code example in Scala too?

Reply:
Not yet.